### PR TITLE
#30 Update examples.rst

### PR DESCRIPTION
**Pull request by OmarElChamaa**

### CODE REVIEW

**Comment by OmarElChamaa** [comment]
I will need to update the link to the github page once @SchawnnDev accepts the pull request

File: source/examples.rst [hunk 1]
```diff
@@ -9,4 +9,12 @@ Text to image GUI
 
 See : `EMF Demos - Text to Image GUI <https://github.com/easy-model-fusion/demos/tree/main/text-to-image-gui>`_
 
-The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
\ No newline at end of file
+The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
+
+
+Text model to image model GUI
```
Comment (Zapharaos) [inline source/examples.rst:15]: not quite sure the title is good enough
makes me think that we are migrating from one model to another through a GUI which doesnt make much sense
however i assume it is a text generation model generates a text which is then used as an input for a text to image model ?

**Review by Zapharaos — CHANGES_REQUESTED** [review]

**Comment by Zapharaos** [comment]
Not linked to any issue btw

File: source/examples.rst [hunk 2]
```diff
@@ -9,4 +9,12 @@ Text to image GUI
 
 See : `EMF Demos - Text to Image GUI <https://github.com/easy-model-fusion/demos/tree/main/text-to-image-gui>`_
 
-The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
\ No newline at end of file
+The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
+
+
+Images from Text Prompts Generated by Text Model GUI
+----------------------------------
```
Comment (Zapharaos) [inline source/examples.rst:16]: have you compiled locally and checked ?
i am not quite sure but if i remember correctly you might encounter an error as the title's text is wider than the line underneath it

File: source/examples.rst [hunk 3]
```diff
@@ -9,4 +9,12 @@ Text to image GUI
 
 See : `EMF Demos - Text to Image GUI <https://github.com/easy-model-fusion/demos/tree/main/text-to-image-gui>`_
 
-The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
\ No newline at end of file
+The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
+
+
+Images from Text Prompts Generated by Text Model GUI
```
Comment (Zapharaos) [inline source/examples.rst:15]: might be more natural to reverse the order ?
i am assuming a user opening the doc searching for a specific example would first be looking at the input and then the result ?

Comment (OmarElChamaa) [reply source/examples.rst]: Title updated

**Review by Zapharaos — APPROVED** [review]

**Pull request merged**